ajax/libs/jquery.form/package.json

```diff
@@ -2,41 +2,38 @@
   "name": "jquery.form",
   "filename": "jquery.form.min.js",
   "title": "jQuery Form Plugin",
-  "description": "A simple way to AJAX-ify any form on your page; with file upload and progress support.",
+  "description": "The jQuery Form Plugin allows you to easily and unobtrusively upgrade HTML forms to use AJAX.",
   "keywords": [
+    "jquery",
+    "ajax",
+    "jquery-plugin",
+    "json",
+    "json-form",
+    "html-form",
     "form",
-    "upload",
-    "ajax"
+    "jquery-form"
   ],
   "version": "3.51",
-  "author": {
-    "name": "M. Alsup",
-    "url": "http://jquery.malsup.com"
+  "homepage": "https://jquery-form.github.io/form/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jquery-form/form.git"
   },
+  "author": "Kevin Morris",
   "licenses": [
     "MIT",
-    {
-      "type": "GPL",
-      "url": "http://malsup.github.com/gpl-license-v2.txt"
-    }
+    "LGPL-3.0"
   ],
-  "homepage": "http://jquery.malsup.com/form/",
-  "docs": "http://jquery.malsup.com/form/",
-  "download": "http://malsup.github.com/jquery.form.js",
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/malsup/form.git",
+    "target": "git://github.com/jquery-form/form.git",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "dist",
         "files": [
-          "jquery.form.js"
+          "**/*"
         ]
       }
```

Comment:
We should also fetch `jquery.js` under `src` directory. Please add the following config:
```
        {
          "basePath": "src",
          "files": [
            "jquery.form*"
          ]
        }
```

Thanks.